charter.md

```diff
@@ -0,0 +1,51 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
+
+The chair of the team is responsible for organizing meetings, driving discussions to a conclusion, and recording decisions. The team appoints an individual as the chair based on consensus once per calendar year. There's no term limit; the same individual can be reappointed as chair.
+
+Meeting time and agenda should be announced by the chair at least 48 hours in advance, and meeting minutes posted in the repository.
+
+The Interop Team's work is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+Substantive changes to this charter are done via the [web-platform-tests RFC process](https://github.com/web-platform-tests/rfcs).
+
+## Scope
+
+The Interop Team's work is metrics designed to measure and improve the interoperability of the web platform.
+
+These metrics are expected to be comprised of:
+
+- Groups of automated tests that can be used to automatically compute an interoperability score for specific platform features in implementations.
+- Work to improve the interoperability of the platform that cannot be measured through test pass rates and is scored by consensus decision.
+
+The team is responsible for maintaining [Interop 2021](https://wpt.fyi/interop-2021), [2022](https://wpt.fyi/interop-2022), [2023](https://wpt.fyi/interop-2023), and defining future interoperability efforts and metrics.
+
+### Out of Scope
+
+This is not a venue for standardization, and work on new platform features is out of scope. However the group may liaise with existing standards groups to help progress work which affects the interoperability of the platform, or the ability to test and measure interoperability of existing features.
+
+Browser features that don't affect web developers, like bookmarks, are out of scope.
+
+## Team Members
+
+The following organizations make up the Interop Team, in alphabetical order:
+
+* Apple
+* Bocoup
+* Google
+* Igalia
+* Microsoft
+* Mozilla
+
+## Joining and Leaving the Team
+
+The Interop Team is made up of organizations with a track record of contributions to the web-platform-tests project.
+
+Organizations may apply to join the team by contacting interop@web-platform-tests.org. New members will be added with a consensus decision of existing members.
```

Comment:
Unless we want to make this explicitly public (which would also be fine), we should be clear that it's not:

```suggestion
Organizations may apply to join the team by contacting interop@web-platform-tests.org. New members will be added with a member-confidential consensus decision of existing members.
```

Reply:
I would be fine with having this be a normal (public) decision or a member-confidential one, with a preference for just making it public.

Reply:
I'm now leaning the other way, to document that this particular decision is member-confidential. @jgraham WDYT?

Reply:
As a point of pedantry (but this is a charter), the _decision_ will be public, the only question is whether any discussions or individual participant positions that go into forming that decision are required to be published. 

At least it seems unlikely that we'd want stronger publication requirements here than for any other work.

Reply:
My main concern is that such language might be wrongly taken to imply that other decision making processes are therefore not "allowed" to be member-confidential, by contrast. 

Also I don't know what the scope "the process" is here; would we be able to publicly minute the fact that FooCorp had applied to join if it was discussed in a meeting? In general does this imply that any and all mention of someone applying to join would be redacted from meeting minutes, or are you only intending it to cover the final details of the consensus-forming process (i.e. which participants objected, in the case the application was not accepted)?

Reply:
What language would you suggest?

Reply:
As for what the words should mean, we should respond in private if the request is private, in public if the request is public, and in either case we should have the option to just relay the outcome without justification or elaboration.

And I guess we don't want to set up that email address, so let's say you contact the chair instead.

Reply:
I don't think any change is needed (apart from fixing the mailing address).